pinot-core/src/main/java/org/apache/pinot/core/plan/TransformPlanNode.java

```diff
@@ -33,13 +34,15 @@
 public class TransformPlanNode implements PlanNode {
   private final Collection<ExpressionContext> _expressions;
   private final ProjectionPlanNode _projectionPlanNode;
+  private boolean _areIdentifiers = true;
```

Comment:
(nit) suggest renaming to `_allExpressionsIdentifiers` or something similar

Reply:
updated!